docs/ossm/ambient/waypoint.md

```diff
@@ -0,0 +1,229 @@
+
+## 1. What is a waypoint proxy for Istio's ambient mode?
+
+A **waypoint proxy** is an optional Envoy-based proxy that provides Layer 7 (L7) processing for specific workloads that are part of a service mesh running in ambient mode. Unlike the traditional sidecar model, where each workload gets its own Envoy proxy, waypoint proxies significantly reduce the number of proxies needed by allowing a single waypoint or set of waypoints to be shared across applications within a similar security boundary (e.g., all workloads in a namespace).
+
+Waypoint proxies operate independently of applications, meaning application owners don't need to be aware of their existence. In **ambient mode**, policies are enforced by the destination waypoint, which acts as a gateway, ensuring all incoming traffic to a resource (namespace, service, or pod) passes through it for policy enforcement.
+
+### 1.1 Do You Need a Waypoint Proxy?
+
+The **ztunnel node proxy** handles most of the features in ambient mode, primarily focusing on Layer 4 (L4) traffic processing. However, if your applications require any of the following L7 mesh functionalities, you'll need to use a waypoint proxy:
+
+* **Traffic Management:** This includes advanced HTTP routing and load balancing, circuit breaking, rate limiting, fault injection, retries, and timeouts.
+* **Security:** For rich authorization policies based on L7 primitives like request type or HTTP headers.
+* **Observability:** To collect HTTP metrics, enable access logging, and implement tracing for your applications.
+
+**Trade-offs:**
+
+* Ambient mode is a newer architecture and may have different operational considerations compared to the traditional sidecar model.
+* L7 features require the deployment of `waypoint` proxies, which add a small amount of overhead for the services that utilize them.
+
+---
+
+## 2. Pre-requisites to Using Waypoint Ambient Mode with OSSM 3
+
+Before installing Istio's ambient mode with OpenShift Service Mesh, ensure the following prerequisites are met:
+
+* **OpenShift Container Platform 4.19+:** This version of OpenShift is required for supported Kubernetes Gateway API CRDs, which are essential for ambient mode functionalities.
+* **OpenShift Service Mesh 3.1.0+ operator is installed:** Ensure that the OSSM operator version 3.1.0 or later is installed on your OpenShift cluster.
+* **Istio deployed in Ambient Mode:** Refer to OSSM Ambient [initial doc](README.md).
+* **Labeling:** Make sure the workloads/namespaces have the appropriate labels for ztunnel traffic redirection.
+
+**Pre-existing Service Mesh Installations:**
```

Comment:
Likewise, this is duplicated from line 40 of [ambient/README.md](https://github.com/openshift-service-mesh/sail-operator/pull/357/files#diff-4193baa5c581fa50bfb0347c67edc90fa0e901ebc3d3882dc8e0b8459e327c98)